File: src/xyz/najlek/elytraprotection/utils/ElytraUtil.java
package xyz.najlek.elytraprotection.utils;

import org.bukkit.inventory.meta.ItemMeta;

import java.util.Arrays;
import java.util.List;

public class ElytraUtil {

    public static ItemMeta transfer(String nick, ItemMeta itemMeta){
        itemMeta.setLore(List.of(nick + "'s Elytra"));
        return itemMeta;
    }

}

File: src/main/kotlin/xyz/najlek/elytraprotection/utils/ElytraUtil.kt
package xyz.najlek.elytraprotection.utils

import org.bukkit.inventory.meta.ItemMeta

object ElytraUtil {
    fun transfer(nick: String, itemMeta: ItemMeta): ItemMeta {
        itemMeta.lore = listOf("$nick's Elytra")
        return itemMeta
    }
}